registers/TUTORIAL.md

```diff
@@ -14,3 +14,45 @@ You can run it with
 
 By default it uses the "register" register (a list of all registers)
 but you can point it at others with the ```--register``` flag.
+
+Step 2
+------
+
+Next, we are going to put the data into a Trillian log. The first step
+is run the log server and create a log.
+
+Build Trillian:
+
+```make trillian```
+
+Run the log server (this will tie up a terminal window):
+
+```make tlserver```
+
+And create a log (if you need to redo this, you'll need to delete
+logid):
+
+```make createlog```
+
+So now we have a log ready for entries from the register. To fill that
+log, we connect to the log using ```grpc.Dial()```, then create a new
+log client with ```trillian.NewTrillianLogClient```.
+
+We then feed each entry as we get it to the log, using
+```QueueLeaf()``` on the log client. The rest is just housekeeping.
```

Comment:
Might be worth noting that the ordering of the log will not be the same as the ordering of the source register?  (At least until the Preordered Log support is present.)